projects/kubernetes/boot.sh

```diff
@@ -16,26 +16,36 @@ set -e
 : ${KUBE_EFI:=}
 : ${KUBE_MAC:=}
 : ${KUBE_CLEAR_STATE:=}
+#: ${KUBE_MASTER_AUTOINIT:=}
+external_data=${KUBE_METADATA}
 
 [ "$(uname -s)" = "Darwin" ] && KUBE_EFI=1
 
 suffix=".iso"
 [ -n "${KUBE_EFI}" ] && suffix="-efi.iso" && uefi="--uefi"
 
+data={
+if [ -n "${external_data}" ] ; then
+	if [ -s "${external_data}" ] ; then
+		external_data=$(tr '\n' ' ' <$external_data | tr -d '\t' )
+	fi
+	data+=$external_data
+fi
+
 if [ $# -eq 0 ] ; then
     img="kube-master"
     # If $KUBE_MASTER_AUTOINIT is set, including if it is set to ""
     # then we configure for auto init. If it is completely unset then
     # we do not.
     if [ -n "${KUBE_MASTER_AUTOINIT+x}" ] ; then
-	kubeadm_data="${kubeadm_data+$kubeadm_data, }\"init\": \"${KUBE_MASTER_AUTOINIT}\""
+	kubeadm_data="${kubeadm_data+$kubeadm_data, }\"init\": {\"content\": \"${KUBE_MASTER_AUTOINIT}\" }"
     fi
     if [ "${KUBE_MASTER_UNTAINT}" = "y" ] ; then
-	kubeadm_data="${kubeadm_data+$kubeadm_data, }\"untaint-master\": \"\""
+	kubeadm_data="${kubeadm_data+$kubeadm_data, }\"untaint-master\": {\"content\": \"\" }"
     fi
 
     if [ -n "${kubeadm_data}" ] ; then
-	data="{ \"kubeadm\": { ${kubeadm_data} } }"
+	data="${data+$data, } \"kubeadm\": {\"entries\": { ${kubeadm_data} } }"
```

Comment:
`$data` is now always at least `{` and in that case I think this will produce invalid JSON since it will be `{, \"kubeadm...`